### PR TITLE
Update DiffEngine to latest

### PR DESCRIPTION
**Pull request by stefannikolei**

### CODE REVIEW

File: Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs [hunk 1]
```diff
@@ -10,16 +10,7 @@ public class SetUpFixture
...
-        DiffTools.AddToolBasedOn(
-            DiffTool.WinMerge,
-            name: "WritableWinMerge",
-            arguments: (temp, target) =>
-            {
-                var leftTitle = Path.GetFileName(temp);
-                var rightTitle = Path.GetFileName(target);
-                return $"/u /wr /e \"{temp}\" \"{target}\" /dl \"{leftTitle}\" /dr \"{rightTitle}\"";
-            }
-        );
```
Comment (belav) [inline Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs:23]: This is used to make `[TestName].test` the writable file and is still needed.

If you add an extra space to `ClassDeclarations.test` and run the test, you want to be able to copy the changes from `ClassDeclarations.actual.test` back to `ClassDeclarations.test`.

**Review by belav — CHANGES_REQUESTED** [review]

File: Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs (hunk 1 above)
Comment (stefannikolei) [reply Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs:23]: ok. I will have a look how to port this arguments to this version

Comment (stefannikolei) [reply Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs]: @belav I tried to achieve this. But on my Mac I could not get the behavior to work. I looked at the changes of DiffEngine and tried to port the arguments. Could you have a look on Windows with an installed version of WinMerge?

Comment (belav) [reply Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs]: This still just has the right side editable.

Is there any need to update? If not it doesn't seem worth the effort to figure out how to make this work on the latest version of DiffEngine.

**Pull request closed without merging**